tools/tcpaccept.py

```diff
@@ -216,6 +222,11 @@
     bpf_text = bpf_text.replace('##FILTER_FAMILY##',
         'if (family != AF_INET6) { return 0; }')
 
+if BPF.kernel_struct_has_field("sock", "sk_protocol") == 1:
```

Comment:
Is the existing offset == 2 fallback still necessary for cases where the sk_protocol field exists, but the absence of BTF information causes the kernel_struct_has_field() check to fail?